package.json

```diff
@@ -24,6 +24,7 @@
   "dependencies": {
     "@actions/core": "^1.10.0",
     "@actions/github": "^5.1.1",
+    "@octokit/request-error": "^5.1.1",
```

Comment:
These two probably will generally upgrade in lock-step since `@actions/github` is a hydrated wrapper around `@octokit`.